Doc/license.rst

```diff
@@ -72,6 +72,18 @@ make these releases possible.
 Terms and conditions for accessing or otherwise using Python
 ============================================================
 
+Python software and documentation are licensed under the
+:ref:`PSF License Agreement <PSF-license>`.
+Starting with Python X.X.X, examples, recipes, and other code in
```

Comment:
I presume X.X.X will be 3.9.0. (Also in the LICENSE file.)

Reply:
I wasn't sure how long this process will take so I didn't want to commit to a particular release.  if it makes it into 3.9.0 that would be great.

Reply:
I would think that this could be backported.  If it is, they should also be listed.  In any case, good idea.

Reply:
I think we can list this as per 3.9.0 even if it doesn't make it in the release (rc2 is scheduled for Sept 14).

Release manager (@ambv) do you think it's reasonable to land this doc-only change?

And I agree we could backport to 3.8.0, in which case we should just say per 3.8.6 (which would be the next release, also going out Sept 14).

Reply:
@gvanrossum Perfectly reasonable.

Reply:
(Both for this to land in 3.8.6rc1 as well as in 3.9.0rc2.)